### ISSUE TITLE
Demos to add to README

### ISSUE THREAD

**Issue by dplarson** [ucsdidea/data_analysis_machine_learning#1]
Demos:

sentiment analysis
- http://nlp.stanford.edu/sentiment/

handwriting generation
- http://www.cs.toronto.edu/~graves/handwriting.cgi

image classification
- http://demo.caffe.berkeleyvision.org/
- https://visual-recognition-demo.mybluemix.net/

image descriptions
- https://developer.aylien.com/text-api-demo?text=&language=auto&tab=image-tags
- http://www.programmableweb.com/news/rapid-rise-deep-learning-computer-vision-technology/analysis/2015/06/19

face substitution
- https://auduno.github.io/clmtrackr/examples/facesubstitution.html
- other: https://github.com/auduno/clmtrackr/

object detector
- https://github.com/inspirit/jsfeat

CNN JS demo
- https://github.com/karpathy/convnetjs

self-organizing map
- http://synaptic.juancazala.com/#/self-organizing-map

Neural Network learning
- http://playground.tensorflow.org/#activation=tanh&batchSize=10&dataset=circle&regDataset=reg-plane&learningRate=0.03&regularizationRate=0&noise=0&networkShape=4,2&seed=0.29667&showTestData=false&discretize=false&percTrainData=50&x=true&y=true&xTimesY=false&xSquared=false&ySquared=false&cosX=false&sinX=false&cosY=false&sinY=false&collectStats=false&problem=classification&initZero=false

Gaze following
- http://gazefollow.csail.mit.edu/demo.html

Project Naptha (extract text from images)
- https://projectnaptha.com/

face tracking
- https://www.beyond-reality-face.com/overview#techdemo-app
- http://ourcodeworld.com/articles/read/151/top-5-best-face-tracking-and-recognition-related-javascript-libraries

colormaps
- http://matplotlib.org/cmocean/
